test_scripts/API/VehicleData/StabilityControlsStatus/030_OnVehicleData_StabilityControlsStatus_ignition_ON_OFF.lua

```diff
@@ -0,0 +1,44 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check receiving StabilityControlsStatus data via OnVehicleData notification
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed
+-- 4) App is activated
+-- 5) App is subscribed on StabilityControlsStatus vehicle data
+--
+-- Steps:
+-- 1) HMI sends VehicleInfo.OnVehicleData notification with StabilityControlsStatus
+--    SDL sends OnVehicleData notification with received from HMI data to App
+-- 2) Perform Ignition OFF/ON
+-- 3) HMI sends VehicleInfo.OnVehicleData notification with StabilityControlsStatus
+--    SDL sends OnVehicleData notification with received from HMI data to App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/commonVehicleData')
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Start SDL, init HMI, connect default mobile", common.start)
+common.Step("Register App1", common.registerApp)
+common.Step("PTU", common.policyTableUpdate, { common.ptUpdate })
+common.Step("Activate App1", common.activateApp)
+common.Step("Subscribe on StabilityControlsStatus VehicleData", common.processRPCSubscriptionSuccess,
+  { "SubscribeVehicleData", { "stabilityControlsStatus" }})
+
+common.Title("Test")
+common.Step("Expect OnVehicleData with StabilityControlsStatus", common.checkNotificationSuccess, 
+	{{ "stabilityControlsStatus" }})
+common.Step("Ignition OFF", common.stopSDL)
+common.Step("Ignition ON", common.start)
+common.Step("Register App1", common.registerAppWOPTU)
```

Comment:
@LitvinenkoIra This registration of application does not resume application. It is needed to perform application registration with application hash.